cpp/example_code/s3/README.md

```diff
@@ -1,54 +1,67 @@
-- [Uploading an object to an Amazon S3 bucket (using a memory buffer instead of local disk copy)](./put_object_buffer.cpp) (PutObject)
-- [Configuring an Amazon S3 bucket for static website hosting](./put_website_config.cpp) (PutBucketWebsite)
-- [Finding, creating, and deleting an Amazon S3 bucket in a sequence](./s3-demo.cpp)
-
-## ⚠ Important
-- We recommend that you grant this code least privilege, or at most the minimum permissions required to perform the task. For more information, see [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege) in the AWS Identity and Access Management User Guide.
-- This code has not been tested in all AWS Regions. Some AWS services are available only in specific [Regions](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
-- Running this code might result in charges to your AWS account. 
-- Running the unit tests might result in charges to your AWS account. [optional]
-
-## Running the Examples
+- [Creating an S3 bucket](./create_bucket.cpp) (CreateBucket)
+- [Deleting an S3 bucket](./delete_bucket.cpp) (DeleteBucket)
+- [Deleting a bucket policy (permission to access resources) of an S3 bucket](./delete_bucket_policy.cpp) (DeleteBucketPolicy)
+- [Deleting an object from an S3 bucket](./delete_object.cpp) (DeleteObject)
+- [Deleting the website configuration of an S3 bucket](./delete_website_config.cpp) (DeleteBucketWebsite)
+- [Getting the access control list (ACL) for an S3 bucket](./get_acl.cpp) (GetBucketAcl)
+- [Getting a bucket policy (permission to access resources) for an S3 bucket](./get_bucket_policy.cpp) (GetBucketPolicy)
+- [Getting an object out of an S3 bucket](./get_object.cpp) (GetObject)
+- [Getting and setting the access control list (ACL) for an S3 bucket](./get_put_bucket_acl.cpp) (GetBucketAcl, PutBucketAcl)
+- [Getting and setting the access control list (ACL) for an object in an S3 bucket](./get_put_object_acl.cpp) (GetObjectAcl, PutObjectAcl)
+- [Getting configuration of an S3 bucket configured for static website hosting](./get_website_config.cpp) (GetBucketWebsite)
+- [Listing all S3 buckets](./list_buckets.cpp) (GetBuckets)
+- [Listing all objects in an S3 bucket](./list_objects.cpp) (ListObjects)
+- [Making requests to S3 across AWS Regions by specifying aws-global as the AWS Region](./list_objects_with_aws_global_region.cpp)
+- [Adding a bucket policy (permission to access resources) to an S3 bucket](./put_bucket_policy.cpp) (PutBucketPolicy)
+- [Uploading an object to an S3 bucket](./put_object.cpp) (PutObject)
+- [Uploading an object to an S3 bucket (asynchronously)](./put_object_async.cpp) (PutObjectAsync)
+- [Uploading an object to an S3 bucket (using a memory buffer instead of local disk copy)](./put_object_buffer.cpp) (PutObject)
+- [Configuring an S3 bucket for static website hosting](./put_website_config.cpp) (PutBucketWebsite)
```

Comment:
(Optional) The new standard is imperative form instead of gerund, to match the titles in the [code library](https://docs.aws.amazon.com/code-library/latest/ug/cpp_1_s3_code_examples.html). Fine to make these changes in a later PR if you like.